operator/src/network/spec.rs

```diff
@@ -57,6 +60,9 @@ pub struct NetworkStatus {
     pub namespace: Option<String>,
     /// Information about each Ceramic peer
     pub peers: Vec<Peer>,
+    /// The number of seconds this network has left to live.
+    /// If unset the network lives forever.
+    pub ttl_seconds_remaining: Option<i64>,
```

Comment:
instead of tracking the number of seconds remaining, can we convert the ttl_seconds into a `expires_at` timestamp first thing at startup, so then we only ever deal in absolute times not relative times?  I generally have found dealing with relative times to be pretty error-prone and easy to mess up, converting to absolute timestamps at the edges as soon as possible tends to help keep things simpler and avoid bugs down the road.